modules/ovms_ai_extension/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2021 Intel Corporation
+
+FROM openvino/model_server:2021.4.1-gpu
```

Comment:
Have you considered using latest OVMS image?

Reply:
We pegged against a specific version for testing/validation purposes. We may update in the future to just use the latest image, but for now we're keeping this.